hypothesis-python/src/hypothesis/internal/reflection.py

```diff
@@ -474,7 +475,7 @@ def check_valid_identifier(identifier):
 
 
 def eval_directory():
-    return storage_directory("eval_source")
+    return os.path.join(hypothesis_home_dir(), "eval_source")
```

Comment:
It appears that this function is tested but never actually used - let's just delete it entirely!

Reply:
I noticed, but I was hesitant to remove it. I'll get right on it.